src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,105 @@ describe("MarketingCollections", () => {
       curatedMarketingCollections: payload,
     })
   })
+
+  it("uses the curated sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CURATED, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockReturnValueOnce(Promise.resolve({ body: payload }))
+
+    const data = await runQuery(query, context)
+
+    expect(context.marketingCollectionsLoader).toHaveBeenCalledWith({
+      size: 2,
+      slugs: [
+        "new-from-tastemaking-galleries",
+        "new-from-nonprofits-acaf27cc-2d39-4ed3-93dd-d7099e183691",
+        "new-from-small-galleries",
+        "new-from-leading-galleries",
+        "new-to-artsy",
+      ],
+    })
+
+    expect(data).toEqual({
+      marketingCollections: payload,
+    })
+  })
+
+  it("passes the params to the marketing collection loader", async () => {
+    const query = gql`
+      {
+        marketingCollections(
+          size: 2
+          category: "trending"
+          slugs: ["percys-z-collection-1", "fiby-z-collection-2"]
+        ) {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
+      },
+    ]
+
+    const context = {
+      authenticatedLoaders: {},
+    } as any
+
+    context.marketingCollectionsLoader = jest
+      .fn()
+      .mockReturnValueOnce(Promise.resolve({ body: payload }))
+
+    const data = await runQuery(query, context)
+
+    expect(context.marketingCollectionsLoader).toHaveBeenCalledWith({
+      size: 2,
+      category: "trending",
+      slugs: ["percys-z-collection-1", "fiby-z-collection-2"],
+    })
+
+    expect(data).toEqual({
+      marketingCollections: payload,
+    })
+  })
+
+  it("throws an error when used with a non-existent category", async () => {
```

Comment:
**suggestion**: Might also be worth adding a test exercising when `sort: CURATED` but category is not provided. I imagine that might exercise the same category missing handling, but we should confirm:

```
No curated sort available for category: undefined
```